Datatable/View/TopActions.php

```diff
@@ -53,6 +53,7 @@ class TopActions extends AbstractViewOptions
     public function __construct()
     {
         $this->options = array();
+        parent::__construct();
```

Comment:
Line 56  calls now `$this->set($this->options);` That does not work because the actions array is required and not set already.

```
    public function configureOptions(OptionsResolver $resolver)
    {
        $resolver->setRequired(array('actions'));

        // ...

        return $this;
    }
```